packaging/dbscripts/clusters_sp.sql

```diff
@@ -595,7 +622,7 @@ CREATE OR REPLACE FUNCTION ClearAllClusterUpgradeRunning ()
 RETURNS VOID AS $PROCEDURE$
 BEGIN
     UPDATE cluster
-    SET upgrade_running = false
+    SET upgrade_running = false, upgrade_progress = 0, upgrade_correlation_id = NULL
```

Comment:
Please reformat to have one column update per line

Reply:
Ack